.github/workflows/web.yml

```
@@ -9,9 +9,9 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v4
+    - uses: actions/checkout@v6
     - name: Use Node.js ${{ matrix.node-version }}
```

actionlint (1.7.12)

[error] property "node-version" is not defined in object type {}

(expression)